From 8e7d6775c212081ba3a516458cbbb994228189e1 Mon Sep 17 00:00:00 2001
From: Chad Dupuis <cdupuis@redhat.com>
Date: Thu, 21 Aug 2014 13:05:03 -0400
Subject: [ethernet] qlcnic: remove a stray semicolon

Message-id: <1408626422-3941-20-git-send-email-cdupuis@redhat.com>
Patchwork-id: 88560
O-Subject: [PATCH RHEL 7.1 019/138] qlcnic: remove a stray semicolon
Bugzilla: 1089352
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Dan Carpenter <dan.carpenter@oracle.com>

Bug 1089352 (https://bugzilla.redhat.com/show_bug.cgi?id=1089352)

net commit id 3cc4a6784dfe26cd1aca55ef28b07eadbc09b960

Just remove a small semicolon.

Signed-off-by: Dan Carpenter <dan.carpenter@oracle.com>
Acked-by: Himanshu Madhani <himanshu.madhani@qlogic.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/qlogic/qlcnic/qlcnic_ethtool.c b/drivers/net/ethernet/qlogic/qlcnic/qlcnic_ethtool.c
index 7761b02..c4a2f5e 100644
--- a/drivers/net/ethernet/qlogic/qlcnic/qlcnic_ethtool.c
+++ b/drivers/net/ethernet/qlogic/qlcnic/qlcnic_ethtool.c
@@ -1711,7 +1711,7 @@ qlcnic_set_dump(struct net_device *netdev, struct ethtool_dump *val)
   qlcnic_dev_request_reset(adapter, val->flag);
   adapter->flags &= ~QLCNIC_FW_RESET_OWNER;
   break;
-;
+
  case QLCNIC_SET_QUIESCENT:
  case QLCNIC_RESET_QUIESCENT:
   state = QLC_SHARED_REG_RD32(adapter, QLCNIC_CRB_DEV_STATE);
-- 
1.7.1